ccd-definition/CaseField.json

```diff
@@ -118,6 +118,57 @@
     "FieldTypeParameter": "Track",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "claimFee",
+    "Label": "Pay fee using Payment by Account (PBA)",
+    "FieldType": "Fee",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "applicantSolicitor1PbaAccounts",
+    "Label": "Select a PBA",
+    "FieldType": "DynamicList",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "noPbaAccountsLabel",
+    "Label": "### You don't have a registered PBA.\n### This case will not be saved.\n## Register an existing PBA with MyHMCTS\n\nIf you want to pay with a different PBA, you or your organisation administrator will need to email MyHMCTSsupport@justice.gov.uk to ask for your PBA to be registered with your MyHMCTS account. You should include your organisation name and PBA number.\n\nIt can then take up to 3 days for your account to be registered. You’ll need to start your claim again to pay the fee.\n## Apply to get a new PBA\n\nYou can [use this form to apply for a new PBA account.](https://www.gov.uk/government/publications/form-fee-account-application-form-fee-account-customer-application-form)\n\nYou’ll need to provide details for you and your organisation, including the required credit limit for your account.\n\nOnce your account has been registered, you’ll need to start your claim again to pay the fee.\n\nRead more information on [registering a PBA.](https://www.gov.uk/guidance/myhmcts-online-case-management-for-legal-professionals)",
+    "FieldType": "Label",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "applicantSolicitor1PbaAccountsIsEmpty",
+    "Label": "Hide pba and show noPbaAccountLabel",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public",
```

Comment:
There was this conversation a while ago about having `_Category` on case fields. I don't think we agreed on approach for all new fields, but this one could be marked as a `WORKAROUND`.

Reply:
I'll be honest I wasn't sure about that from the get go, I think if naming and comments can explain a field sufficiently then that should do. However if @AlisonRevittHM asks for it then I will add.

Reply:
This can always be retrofitted as well, has no functional effect for the story